packages/frontend/src/components/RenderedSql.tsx

```diff
@@ -43,6 +43,18 @@ export const RenderedSql = () => {
         [language],
     );
 
+    const formattedSql = useMemo(() => {
+        if (!data?.query) return '';
+        try {
+            return format(data.query, {
+                language: getLanguage(project?.warehouseConnection?.type),
+            });
+        } catch (e) {
+            console.error('Error rendering SQL:', e.message || 'Unknown error occurred');
```

Comment:
wonder if it needs it's own error instance logged in case we need to tackle some cases but I'm happy with the change and we do the same in `sqlRunnerSlice.ts`